roles/app/templates/application.yml.j2

```diff
@@ -24,15 +24,13 @@ ADMIN_PASSWORD: {{ admin_password }}
 {% if enable_rails_apm is defined %}
 DATADOG_RAILS_APM: {{ enable_rails_apm }}
 {% endif %}
+
 {% if s3_access_key is defined %}
 S3_ACCESS_KEY: {{ s3_access_key }}
 {% endif %}
 {% if s3_secret is defined %}
 S3_SECRET: {{ s3_secret }}
 {% endif %}
-{% if s3_region is defined %}
-S3_REGION: {{ s3_region }}
```

Comment:
this is fine as long as we change Katuma's secrets to use `s3_backups_region` instead as that it's intended use (I'll do that now).

Reply:
Done in https://github.com/openfoodfoundation/ofn-secrets/pull/42